modules/mono/glue/Managed/Files/Mathf.cs

```diff
@@ -261,6 +261,13 @@ public static real_t Sinh(real_t s)
             return (real_t)Math.Sinh(s);
         }
 
+        public static real_t SmoothStep(real_t from, real_t to, real_t weight)
+        {
+            if(IsEqualApprox(from, to)) return from;
```

Comment:
Style issue here I guess, should have a space after `if`.

Reply:
Fixed